contracts/data/WitnetBoardDataACLs.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity >=0.7.0 <0.9.0;
+
+import "./WitnetBoardData.sol";
+
+/**
+ * @title Witnet Access Control Lists storage layout, for centralized request boards.
+ * @author Witnet Foundation
+ */
+abstract contract WitnetBoardDataACLs is WitnetBoardData {  
+  struct SWitnetBoardACLs {
+    mapping (address => bool) isReporter_;
+  }
+
+  constructor() {
+    __acls().isReporter_[msg.sender] = true;
+  }
+
+  modifier onlyReporters {
+    require(__acls().isReporter_[msg.sender], "WitnetBoardDataACLs: unauthorized reporter");
+    _;
+  }  
+
+  function __acls() internal pure returns (SWitnetBoardACLs storage _struct) {
```

Comment:
could you please add comments to this functions so that are easier to understand?